src/assets/scripts/client/ui/SettingsController.js

```diff
@@ -50,7 +63,8 @@ export default class SettingsController {
      * @chainable
      */
     init() {
-        const $options = $(UI_SETTINGS_MODAL_TEMPLATE);
+        this.$dialog = $(UI_SETTINGS_MODAL_TEMPLATE);
+        this.$body = this.$dialog.find('.dialog-body');
```

Comment:
do we have `.dialog-body` in the `SELECTORS` constant?

Reply:
also, instead of `$body` give this a more contextual name.  at first glance, I thought this was referencing the document `body` and not the `dialog-body`.

Reply:
done